### PR TITLE
Cortex XDR content pack

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Description
Cortex XDR content pack initial commit

### CODE REVIEW

**Comment by Itay4** [comment]
https://github.com/demisto/etc/issues/19633

File: Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_test.yml [hunk 1]
```diff
@@ -0,0 +1,357 @@
+id: Test XDR Playbook
```
Comment (anara123) [inline Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_test.yml]: I think this should be in TestPlaybooks folder.

Comment (Itay4) [reply Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_test.yml]: @anara123 see discussion in the issue

**Comment by Itay4** [comment]
> In README.md use documentation from [demisto/etc#18211](https://github.com/demisto/etc/issues/18211)
> Add .secrets-ignore

will add doc
there is `.secrets-ignore.`

File: Packs/CortexXDR/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1 @@
+## [Unreleased]
```
Comment (yaakovi) [inline Packs/CortexXDR/CHANGELOG.md]: currently this file is automatically generated for new content entities (in the clear release notes process).

will it be included in the packs from day one or from the first modification?

**Comment by Itay4** [comment]
@glicht @anara123 @yaakovi can we merge this ?

**Comment by yaakovi** [comment]
failed on secrets (false positive)

**Pull request merged**